### ISSUE TITLE
Provide option to attach debugger

### ISSUE THREAD

**Issue by hznut** [ashiina/lambda-local#107]
Eg. `lambda-local --debug-brk -l index.js -h handler -e event-samples/s3-put.js`

**Comment by musicin3d** [comment]
@gpotter2 Thank worked for me. Thanks! I'm not sure what it would take to support this officially, but that is a pretty simple solution.

**Comment by gpotter2** [comment]
@musicin3d I know that's would be pretty nice to have it integrated, but I have not found a way to call the inspector from inside nodejs (this way calls it while loading nodejs)...

**Comment by musicin3d** [comment]
@gpotter2 Agreed. I suppose you could spawn a child process if debugging flags are present. Is that a bad hack?
